src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($box-side, 1fr));
+
+  @media(min-width: $box-side * $desired-col-num) {
+    grid-template-columns: repeat($desired-col-num, minmax($box-side, 1fr));
+  }
+
+  &__box {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
let's not use flex

Reply:
Is centering content with flex a bad practice when the parent element has `display: grid`?